Components/toast/toast.css

```diff
@@ -0,0 +1,17 @@
+.toast-box{
+    display: flex;
+    background-color: rgb(26, 141, 187);
+    padding: 0.75rem 2rem;
+    border-radius: 5px;
+    color: white;
+    font-weight: 500;
+}
+

```

Comment:
adviced to use a variable for bg color if using regularly and avoid using the color names directly

Reply:
I made this file, seperately but i'll try to implement this in the downloadable css file. I'll change the white to rgb aswell.